lib/src/license.dart

```diff
@@ -17,15 +17,16 @@ Future<List<LicenseFile>> detectLicensesInDir(String baseDir) async {
   final rootFiles = await new Directory(baseDir).list().toList();
   final licenseCandidates = rootFiles
       .where((fse) => fse is File)
+      .cast<File>()
```

Comment:
I believe the recommendation is to replace these two lines with a single:
```dart
  .whereType<File>()
```

Reply:
I've changed it, but getting

````
  UnimplementedError: whereType is not yet supported
  dart:collection/list.dart 198:7               _ListBase&Object&ListMixin.whereType
  package:pana/src/license.dart 19:8            detectLicensesInDir
````

Changing it back for now.

Reply:
Need to wait until preview-dart-2 is enabled everywhere...